homeassistant/components/overkiz/manifest.json

```diff
@@ -13,7 +13,7 @@
   "integration_type": "hub",
   "iot_class": "cloud_polling",
   "loggers": ["boto3", "botocore", "pyhumps", "pyoverkiz", "s3transfer"],
-  "requirements": ["pyoverkiz==1.10.1"],
+  "requirements": ["pyoverkiz==1.9.0"],
```

Comment:
~~
Revert this change.

As 2023.9.0 made an unwanted breaking change (see) and .1 did not revert the revert made for .0 (see https://github.com/home-assistant/core/pull/99741#pullrequestreview-1613582253) we should now let it like that.

To fix the unique_id breaking change, see #99765.

With that, I request changes.
~~

Changed my mind:
This line will fix the issue #99404
Then the unique_id breaking change will be limited to 2023.9.0 only.
So people upgrading from <= 2023.8.x to 2023.9.1 won't have breaking change.

I don't know if it's nice doing like that.

I didn't dare block (request change) the merge for 2023.9.0  but maybe I should have.

With that, I approve.

Reply:
We can just add an alert to prevent users for 2023.9.0 breaking change and advice to skip .0 and go directly to 2023.9.1+ !

Reply:
What will happen for people going from 2023.9.0 to 23023.9.1 then now?

Reply:
Chatted by Discord:

Getting their ID back from before 2023.9.0

Going from <= 2023.8.x to >= 2023.9.1 won't notice any issue.

Not sure but maybe people that upgraded to 2023.9.0 that did not customize anything to the entity aren't also concerned.

Thanks for the merge 🥳
So now the breaking change only concerns certain persons on 2023.9.0 only !